### DISCUSSION TITLE
Why URL regex throw error while `vet`.

### DISCUSSION THREAD

**Discussion by BhautikChudasama** [cue-lang/cue#2156] [Q&A]
CUE throws an error while validating the URL.
Here is my check.cue file:
```cue
#Endpoint: ~= /^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$/
endpoints: [...#Endpoint]
```
I am checking with `config.yaml`
```yaml
endpoints:
- "https://exmple.com"
- "https://example1.com"
```

It throws an error:
```log
$> cue vet config.yaml check.cue
unknown escape sequence:
    ./config.cue:1:35
unknown escape sequence:
    ./config.cue:1:55
unknown escape sequence:
    ./config.cue:1:68
unknown escape sequence:
    ./config.cue:1:110
```

**Comment by mvdan** [comment]
This doesn't look like a bug, and you didn't use the bug report template, so I'm moving this to the Q&A section.

You can use the playground to share code. For example: https://cuelang.org/play/?id=R4RXnpdf-7M#cue@export@cue

It looks like your syntax wasn't right. If you look at https://cuelang.org/docs/references/spec/#comparison-operators, you'll see that the operator is `=~`. And your string isn't quoted, which isn't a good idea when it contains so many special characters. This appears to work: https://tip.cuelang.org/play/#cue@export@cue
